pages/common/feroxbuster.md

```diff
@@ -0,0 +1,20 @@
+# feroxbuster
+
+> Brute-forces hidden paths on web servers and more.
```

Comment:
```suggestion
> Simple, fast, recursive content discovery tool written in Rust.
> Used to brute-force hidden paths on web servers and more.
```
Suggested change: I think this would be better.